cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -216,6 +247,32 @@ public LogSaverTableUtil getLogSaverTableUtil(DatasetFramework dsFramework,
 
         @Provides
         @Singleton
+        @Named("scheduleStoreTableUtil")
```

Comment:
how come there are all these provider methods? Shouldn't things be bound already in the modules already in use?